### PR TITLE
[Deprecation Management] Detonate URL - Generic playbook fix

### PR DESCRIPTION
**Pull request by samuelFain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: https://jira-hq.paloaltonetworks.local/browse/CIAC-4886

## Description
This PR is a fix for [[Deprecation Management] Detonate URL - Generic playbook](https://github.com/demisto/content/pull/24349)  PR.
Regarding `Detonate URL - Generic` playbook.
___
In the original [[Deprecation Management] Detonate URL - Generic playbook](https://github.com/demisto/content/pull/24349)  PR , The `Detonate URL - CrowdStrike Falcon Intelligence Sandbox` sub-playbook was removed by mistake instead of the deprecated `Detonate URL - CrowdStrik` sub-playbook.
This PR added back the `Detonate URL - CrowdStrike Falcon Intelligence Sandbox` sub-playbook and removed the deprecated `Detonate URL - CrowdStrik` sub-playbook, as intended. 

## Screenshot
Detonate_URL_-_Generic_Tue_Feb_21_2023

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_40.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detonate URL - Generic
+
+- Fixed an issue where the wrong sub-playbook was removed from the playbook.
```
Comment (ShacharKidor) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_40.md:6]:
```suggestion
- Fixed an issue where the **Detonate URL - CrowdStrike Falcon Intelligence Sandbox** sub-playbook was unnecessarily removed.
- Removed the deprecated **Detonate URL - CrowdStrike** sub-playbook. 
```

File: Packs/CrowdStrikeFalconX/pack_metadata.json [hunk 2]
```diff
@@ -22,4 +22,4 @@
     "marketplaces": [
         "xsoar"
     ]
-}
\ No newline at end of file
+}
```
Comment (ShacharKidor) [inline Packs/CrowdStrikeFalconX/pack_metadata.json:25]: Please revert this change.

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
@samuelFain - Nice :)

File: Packs/CrowdStrikeFalconX/pack_metadata.json (hunk 2 above)
Comment (samuelFain) [reply Packs/CrowdStrikeFalconX/pack_metadata.json]: Done.

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_40.md (hunk 1 above)
Comment (samuelFain) [reply Packs/CommonPlaybooks/ReleaseNotes/2_3_40.md]: Done.

**Pull request merged**